.github/labeler.yml

```diff
@@ -0,0 +1,56 @@
+sig/engine:
+  - "cmd/*"
+  - "components/backup/*"
+  - "components/collections/*"
+  - "components/engine_*/*"
+  - "components/external_storage/*"
+  - "components/file_system/*"
+  - "components/keys/*"
+  - "components/raft_log_engine/*"
+  - "components/raftstore/*"
+  - "components/rusoto_util/*"
+  - "components/sst_importer/*"
+  - "components/test_raftstore/*"
+  - "components/test_sst_importer/*"
+  - "components/tikv_util/*"
+  - "etc/*"
+  - "metrics/*"
+  - "src/import/*"
+  - "src/server/*"
+  - "src/config.rs"
+  - "src/lib.rs"
+  - "src/read_pool.rs"
+
+sig/raft:
+  - "components/batch-system/*"
+  - "components/cdc/*"
+  - "components/configuration/*"
+  - "components/encryption/*"
```

Comment:
`components/encryption` should be under sig/engine

Reply:
Can you modify my pr directly? I'm not very familiar with the tikv codebase.